core/src/main/scala/chisel3/experimental/hierarchy/core/Lookupable.scala

```diff
@@ -40,6 +58,24 @@ trait Lookupable[-B] {
   def definitionLookup[A](that:     A => B, definition: Definition[A]): C
   protected def getProto[A](h:      Hierarchy[A]): A = h.proto
   protected def getUnderlying[A](h: Hierarchy[A]): Underlying[A] = h.underlying
+
+  // Since method that may eventually replace instanceLookup and definitionLookup.
```

Comment:
not sure about this comment's english... what does "Since method" mean?

Reply:
It's a typo, meant "single`, will fix.